src/css/style.css

```diff
@@ -30,6 +30,21 @@ a:hover {
     color: #333;
 }
 
+footer {
+    position:absolute;
```

Comment:
The CSS should be formatted just like the other blocks:

```

Reply:
We should fix this issue (https://github.com/AlicanteFrontend/alicantefrontend.github.io/issues/22) to avoid this in the future.